similar/main/multibot.cpp

```diff
@@ -1033,7 +1033,7 @@ void multi_do_boss_teleport(const playernum_t pnum, const ubyte *buf)
 
 	digi_link_sound_to_pos( Vclip[VCLIP_MORPHING_ROBOT].sound_num, teleport_segnum, 0, boss_obj->pos, 0 , F1_0);
 	digi_kill_sound_linked_to_object( boss_obj);
-	digi_link_sound_to_object2(SOUND_BOSS_SHARE_SEE, boss_obj, 1, F1_0, vm_distance{F1_0*512});	//	F1_0*512 means play twice as loud
+	boss_link_see_sound(boss_obj);
```

Comment:
This is a minor semantic change in Descent 2.  Previously, the code always used `SOUND_BOSS_SHARE_SEE` (defined as `183`); now it will use the sound specified in the robot_info structure.  This is probably a change for the better, but I felt I should point out that it is a change.

If I did the analysis correctly, the boss robot info specifies these see sounds:

Descent 1: First Strike, Level 7: 170
Descent 1: First Strike, Final: 170
Descent 2: Counterstrike, Level 4: 183
Descent 2: Counterstrike, Level 8: 225
Descent 2: Counterstrike, Level 12: 186
Descent 2: Counterstrike, Level 16: 192
Descent 2: Counterstrike, Level 20: 237
Descent 2: Counterstrike, Level 24: 195
Descent 2: Vertigo, first boss: 195
Descent 2: Vertigo, second boss: 47